rest_docs/rest-catalog-open-api.yaml

```diff
@@ -29,34 +29,39 @@ info:
     Defines the specification for the first version of the REST Catalog API.
     Implementations should ideally support both Iceberg table specs v1 and v2, with priority given to v2.
 servers:
-  - url: "{scheme}://{host}/{basePath}"
+  - url: '{scheme}://{host}/{basePath}'
```

Comment:
can we have a separated PR just for this to fix the quotation and other unrelated stuffs?

Reply:
Oh yeah. I don't even know how those changes got into this branch. I'll definitely revert them 👍

Reply:
These have been removed. Sorry about that!